### DISCUSSION TITLE
blog/reflectionfirstof2023

### DISCUSSION THREAD

**Discussion by giscus** [khj0426/HJ_Devlog#3] [General]
# blog/reflectionfirstof2023

개발관련 여러 지식을 기록하는 공간

https://hj-devlog.vercel.app/blog/reflectionfirstof2023

**Comment by wukdddang** [comment]
효중님 안녕하세요! 4기 우창욱입니다. 지금까지 공부하신 내용들과 느낀점들이 잘 녹아들어 있는 것 같아 술술 읽혔습니다 ㅎㅎ 그리고 블로그가 잘 정리되어 있어서 저도 제 블로그를 잘 가꿔봐야겠다는 자극을 받고 갑니다! ㅎㅎ 남은 기간도 화이팅입니다! 🔥🔥

**Reply by khj0426** [reply]
창욱님!!! 반가워요!!! 창욱님 블로그 잘 보고 있어요! 실행 컨텍스트에 관해 쓴 글 학습하는데 도움이 많이 되었습니당 남은 기간 진짜 화이팅해봐요 💪💪💪💪
